modules/ROOT/pages/sync/data_sources.adoc

```diff
@@ -0,0 +1,58 @@
+include::{partialsdir}/attributes.adoc[]
+
+//':context:' is a vital parameter. See: http://asciidoctor.org/docs/user-manual/#include-multiple
+:context: data_sources
+
+[id='{context}_data_sources']
+= Managing your data sources using the {sync-service} UI
+
+== Data Source Introduction
+
+A Data Source is a representation of your data storage solution. The Data Source consists of a *Name*, a *Type*, and a *Configuration*. The *Type* of Data Source which are currently supported are *Postgres* and *InMemory*. For information on how to use *Postgres*, refer to the documentation https://www.postgresql.org/docs[here]. The *InMemory* Data Source that is currently supported is *NeDB*, for more information on how to use this please refer to the the documentation https://github.com/louischatriot/nedb[here]. *Note* that the *InMemory* Data Source is not recommended for production use.
```

Comment:
InMemory -> In Memory

Reply:
Postgres is not currently supported but i think it's ok to mention it here already as it will be the next data source to add.

Reply:
yep, I added it based on the fact we will be doing it